pkg/apis/compliance/v1alpha1/tailoredprofile_types.go

```diff
@@ -6,6 +6,14 @@ import (
 
 // FIXME: move name/rationale to a common struct with an interface?
 
+type TailoredProfileOutputType string
```

Comment:
Pro tip: if you add `// +kubebuilder:validation:Enum=ConfigMap` then kubebuilder would generate a validator that only accepts the correct values on its own.

Reply:
Oops, you did this in a subsequent patch :)